### PR TITLE
handler::queue: add a test

### PR DESCRIPTION
**Pull request by mexus**
Add a unit test for a `Queue`

### CODE REVIEW

**Comment by rossnomann** [comment]
Related issue: #33

File: src/handler/queue.rs [hunk 1]
```diff
@@ -46,3 +46,80 @@ impl Queue {
...
+        counter: Arc<AtomicUsize>,
+    }
+
+    impl UpdateHandler for SimpleHandler {
+        fn handle(&mut self, _: Update) {
+            self.counter.fetch_add(1, Ordering::SeqCst);
+        }
+    }
+
+    fn make_message() -> Message {
```
Comment (rossnomann) [inline src/handler/queue.rs]: Use serde_json please
Example: https://github.com/tg-rs/carapax/blob/master/src/handler.rs#L410

Comment (rossnomann) [reply src/handler/queue.rs]: Update: use json! macro. See #38

**Comment by mexus** [comment]
As discussed, closing this PR in favour of using a lazy queue from an external crate.

**Pull request closed without merging**